politeiawww/api/www/v1/v1.go

```diff
@@ -56,6 +56,7 @@ const (
 	RouteEditProposal             = "/proposals/edit"
 	RouteAuthorizeVote            = "/proposals/authorizevote"
 	RouteStartVote                = "/proposals/startvote"
+	RouteMessageProposer          = "/proposals/message"
```

Comment:
I think this should be a user route instead of a proposal route since you're messaging a user.